### PR TITLE
Chunter: Download file action

### LINKED ISSUE #1525: Chunter: Download file action

### PR DESCRIPTION
**Pull request by denis-bunakalya**
closes https://github.com/hcengineering/anticrm/issues/1525

### CODE REVIEW

File: plugins/attachment-resources/src/components/icons/FileDownload.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--
+// Copyright © 2022 Anticrm Platform Contributors.
```
Comment (BykhovDenis) [inline plugins/attachment-resources/src/components/icons/FileDownload.svelte]: It's extra, use only `// Copyright © 2022 Hardcore Engineering Inc.`

File: plugins/attachment-resources/src/components/AttachmentActions.svelte [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<!--
+// Copyright © 2022 Anticrm Platform Contributors.
```
Comment (BykhovDenis) [inline plugins/attachment-resources/src/components/AttachmentActions.svelte]: It's extra, use only `// Copyright © 2022 Hardcore Engineering Inc.`

File: plugins/chunter-resources/src/components/EditChannelDescriptionAttachments.svelte [hunk 3]
```diff
@@ -65,6 +65,9 @@
             <AttachmentPresenter value={attachment} />
           </div>
           <div class="eAttachmentRowActions" class:fixed={i === selectedRowNumber}>
+            <a href={getFileUrl(attachment.file)} download={attachment.name}>
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/components/EditChannelDescriptionAttachments.svelte:68]: Could you please add some margins
изображение

**Review by BykhovDenis — CHANGES_REQUESTED** [review]

File: plugins/attachment-resources/src/components/AttachmentActions.svelte (hunk 2 above)
Comment (denis-bunakalya) [reply plugins/attachment-resources/src/components/AttachmentActions.svelte]: Done

File: plugins/attachment-resources/src/components/icons/FileDownload.svelte (hunk 1 above)
Comment (denis-bunakalya) [reply plugins/attachment-resources/src/components/icons/FileDownload.svelte]: Done

File: plugins/chunter-resources/src/components/EditChannelDescriptionAttachments.svelte (hunk 3 above)
Comment (denis-bunakalya) [reply plugins/chunter-resources/src/components/EditChannelDescriptionAttachments.svelte:68]: Done

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**